roles/common/defaults/main.yml

```diff
@@ -14,3 +14,4 @@ collectd_disk_ose_registry: "{{ customer_collectd_disk_ose_registry | default(No
 docker_storage_dev: "{{ customer_docker_storage_dev | default('vdb') }}"
 common_logstash_containers: "{{ logstash_containers | default('false') }}"
 rcip_openshift_scripts_version: HEAD
+cloud_persist_hostname_disabled: "{{ customer_cloud_persist_hostname_disabled | default('false') }}"
```

Comment:
In order to make it more generic, maybe we should by default not apply modification and add a variable to say, perform action on cloudinit config ? And a second for the value of preserve_hostname
Something like that :

```
cloud_init_manage: {{ customer_cloud_init_manage | default('false') }}"
cloud_init_preserve_hostname: {{ customer_cloud_init_preserve_hostname | default('true') }}"
```